src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/blackwing_lair.h

```diff
@@ -59,7 +59,7 @@ enum CreatureIds
 enum GameObjectIds
 {
     GO_BLACK_DRAGON_EGG         = 177807,
-    GO_BOSSGATE01               = 175946,
+    GO_BOSSGATE01               = 176965,
```

Comment:
What is the correct name? Don't have the DB installed and, to be honest, I don't have enough drive space to re-install everything...